platform/go/deps.bzl

```diff
@@ -7,7 +7,8 @@ def go_dependencies():
 
     # TODO: stop using binaries built by upstream; use our own
     go_register_toolchains(
-        go_version = "1.12.10",
+        # TODO: might need to bump kubernetes to support this newer version
```

Comment:
what does this comment mean?

Reply:
Kubernetes requires being locked to a particular Go version, and we've previously run into trouble caused by using _too new_ of a version of Go with a version of kubernetes.

So we should probably upgrade kubernetes to a version that's appropriate for our new version of Go.